includes/template.php

```diff
@@ -28,6 +28,8 @@
 
 // Build the URL to do the request to the Frontity server.
 $url = $frontity_server . $_SERVER['REQUEST_URI'];
+// Add the `frontity_embedded` query.
+$url .= ( parse_url( $url, PHP_URL_QUERY ) ? '&' : '?' ) . 'frontity_embedded=true';
```

Comment:
I would use here [`wp_parse_url`](https://developer.wordpress.org/reference/functions/wp_parse_url/):

```suggestion
$url .= ( wp_parse_url( $url, PHP_URL_QUERY ) ? '&' : '?' ) . 'frontity_embedded=true';
```